tests/Http/Offer/UpdateWorkflowStatusRequestHandlerTest.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Offer;
+
+use CultuurNet\UDB3\Event\Commands\Moderation\Approve;
+use CultuurNet\UDB3\Event\Commands\Moderation\Publish;
+use CultuurNet\UDB3\Event\Commands\Moderation\Reject;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Offer\Commands\DeleteOffer;
+use CultuurNet\UDB3\StringLiteral;
+use Iterator;
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\SchemaError;
+use PHPUnit\Framework\TestCase;
+
+final class UpdateWorkflowStatusRequestHandlerTest extends TestCase
+{
+    use AssertApiProblemTrait;
+
+    private TraceableCommandBus $commandBus;
+    private UpdateWorkflowStatusRequestHandler $updateWorkflowStatusRequestHandler;
+
+    protected function setUp(): void
+    {
+        $this->commandBus = new TraceableCommandBus();
+        $this->commandBus->record();
+        $this->updateWorkflowStatusRequestHandler = new UpdateWorkflowStatusRequestHandler($this->commandBus);
+    }
+
+    /**
+     * @test
+     */
+    public function it_dispatches_no_command_for_workflow_status_draft(): void
+    {
+        $request = (new Psr7RequestBuilder())
+            ->withRouteParameter('offerType', 'events')
```

Comment:
Would it make sense to also test the `place` endpoint to make sure that the if-statement to create the correct command works properly?

Reply:
It kind of hurts the readability of that data provider because I need to prefix the keys of the `yield` statement so they remain unique then. But I've implemented it as well as data providers to run the other tests for both events & places in https://github.com/cultuurnet/udb3-backend/pull/1238/commits/7d9c2f598909010790ac1ac8f355e077d37b1107